tools/generate-sqlite-files/.gitignore

```
@@ -0,0 +1,3 @@
+allfac*
+*.zip
+*.sqlite3
```

```suggestion
*.sqlite3

```
Can you add a newline at the eof, just to remove the `-` at the end?